build.sbt

```diff
@@ -21,6 +21,7 @@ sourceDistName := "apache-pekko-grpc"
 sourceDistIncubating := true
 ThisBuild / versionScheme := Some(VersionScheme.SemVerSpec)
 ThisBuild / resolvers += Resolver.ApacheMavenSnapshotsRepo
+ThisBuild / resolvers += Resolver.ApacheMavenStagingRepo
```

Comment:
How is this related to this PR?

Reply:
needed to fix the build - the mima check is finding the v1.0.2-RC1 tag and trying to pull the 1.0.2-RC1 jars

Reply:
Ah I see